.travis.yml

```diff
@@ -1,6 +1,9 @@
 os: linux
 language: scala
 
+jdk:
+  - openjdk8
+
 script:
 - curl https://raw.githubusercontent.com/scala-native/scala-native/master/scripts/travis_setup.sh | bash -x
```

Comment:
This is not needed anymore. Set 
```
dist: bionic
```
too.

Reply:
ah, good to know. I'll just copy-paste the one you mentioned verbatim

Reply:
Btw, why is `__.__.test` needed, instead of `__.test`? Isn't a double __ redundant?

Reply:
Yes, you are right.